lib/grpnice.gi

```diff
@@ -836,6 +836,13 @@ end);
 AttributeMethodByNiceMonomorphism( Size,
     [ IsGroup ] );
 
+#############################################################################
+##
+#M  StructureDescription( <G> )
+##
+AttributeMethodByNiceMonomorphism( StructureDescription,
```

Comment:
So what does this differently than what is already in ````grpnames.gi:1914-1924````? Is this not superfluous?